procedures: edit the sections about the che-editor.yaml file (#2466)
codenvy/che-docs: modules/end-user-guide/pages/selecting-an-in-browser-ide-for-one-new-workspace.adoc

:_content-type: PROCEDURE
:description: Selecting an in-browser IDE for one new workspace
:keywords: select-IDE
:navtitle: Selecting an in-browser IDE for one new workspace
:page-aliases: selecting-an-ide-by-using-a-url-parameter.adoc

[id="selecting-an-in-browser-ide-for-one-new-workspace_{context}"]
= Selecting an in-browser IDE for one new workspace

> You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository and yet with the personal choice of the in-browser IDE.

Reviewer: ```suggestion
You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository but use their preferred in-browser IDE.
```

Author: :) I'm trying to avoid using `their` here. I'm deleting `yet with` to improve readability here, so it is rewritten like this now:
```Suggestion
You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository and the personal choice of the in-browser IDE.
```

.Procedure

> . Include the URL parameter for the workspace IDE in the URL for starting a new workspace. See xref:url-parameter-for-the-workspace-ide.adoc[].

Reviewer: Is it literally a special start-a-new-workspace The wording is a bit confusing but I'm not sure how to make it better.

Author: The wording is very use-case specific:

_Working with Che in your browser involves multiple URLs:_
    _The URL of your organization’s Che instance, used as part of all the following URLs_
    _The URL of the Workspaces page of your Che dashboard with the workspace control panel_
    _**The URLs for starting a new workspace**_
    _The URLs of your workspaces in use_

Source: https://www.eclipse.org/che/docs/stable/end-user-guide/starting-a-new-workspace-with-a-clone-of-a-git-repository/

> . Include the xref:url-parameter-for-the-workspace-ide.adoc[URL parameter for the workspace IDE] in the xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[URL for starting a new workspace].

Reviewer 2: Do you think giving this xref is the best approach for the user? 
Will _most_ users know how to include the URL parameter just by reading this step? If not, maybe it would be better to provide additional information about the URL parameter here so most of them don't have to go looking for it.

Author: Indeed. This is a basic step that is already documented and that a user has to know how to perform. I resisted an SME's suggestion to move this xref into `.Additional resources`. I'm only including it here to avoid ambiguity for readers (in case they might think it's something different). I'd like to avoid documenting this step twice in two different locations in the docs as a docs philosophy to minimize the amount of content as well as reduce the overhead of maintaining the docs.

> . Visit the URL in the browser. See xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[].

Reviewer 3: I would reference "xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[]." to related links rather than in the procedure.

Author: @l0rd, WDYT? Just in-line hyperlinks, without `See ...` or `Additional resources`.
![image](https://user-images.githubusercontent.com/78564304/195542792-754f3625-db3f-4145-8447-75f89882b890.png)

> . Visit the URL in the browser. See xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[].

Reviewer: ```suggestion
. Open the URL in the browser. See xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[].
```

Author: Thanks for the tip, Jana. I'll keep this option in mind for future use. However, in this case, there's some change of the URL, so the user ends up getting a different URL open up in the browser tab.
